### DISCUSSION TITLE
[@types/react-dom] Testing utils Simulate all pointer events missing

### DISCUSSION THREAD

**Discussion by AndyOGo** [DefinitelyTyped/DefinitelyTyped#64738] [Issues with a @types package]
`react-dom@17.0.2` Testing utils supports also pointer events.

> `Simulate` has a method for [every event that React understands](https://reactjs.org/docs/events.html#supported-events).
> https://reactjs.org/docs/test-utils.html#simulate

https://github.com/facebook/react/blob/12adaffef7105e2714f82651ea51936c563fe15c/packages/react-dom/src/test-utils/ReactTestUtils.js#L616-L702

[a comment by an automated account is not shown]

**Reply by AndyOGo** [reply]
@MartynasZilinskas, @theruther4d, @Jessidhia, @eps1lon.

**Comment by eps1lon** [comment]
Thank you for opening this issue. I'm happy to review a PR adding pointer event types to `react-dom/test-utils`. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5a94716c6788f654aea7999a5fc28f4f1e7c48ad/types/react-dom/test-utils/index.d.ts#L85 is probably a good starting point.

**Reply by AndyOGo** [reply]
Great @eps1lon  I finally found time to do it :)
PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65261
